### PR TITLE
Improvement: Add intellij 2021.3 support

### PR DESCRIPTION
**Pull request by robert3005**
In intellij 2021.3 (currently in EAP) the `SerializableHasSerialVersionUidFieldInspection` has changed the package. I had added new location to the set of the ones being search when initializing.

### CODE REVIEW

**Comment by vanbeekb** [comment]
Please consider getting this implemented ASAP as the plugin will become unusable once the new version of IntelliJ is released.

**Comment by visajkin** [comment]
@dubreuia Any updates on support for 2021.3?

**Comment by BumbrT** [comment]
@dubreuia

**Review by dubreuia — APPROVED** [review]

**Comment by sfragata** [comment]
Hi @robert3005, there was a check fail

**Comment by robert3005** [comment]
Sure, I think those were related to previous EAP changes but I can fix those

File: src/main/java/com/intellij/codeInspection/visibility/CustomAccessCanBeTightenedInspection.java [hunk 1]
```diff
@@ -96,7 +98,7 @@ public boolean isEnabledByDefault() {
   @Override
   @NotNull
   public String getGroupDisplayName() {
-    return GroupNames.VISIBILITY_GROUP_NAME;
+    return InspectionsBundle.message("group.names.visibility.issues");
```
Comment (robert3005) [inline src/main/java/com/intellij/codeInspection/visibility/CustomAccessCanBeTightenedInspection.java:101]: not sure which version this got introduced in but looks pretty old overall

Comment (robert3005) [reply src/main/java/com/intellij/codeInspection/visibility/CustomAccessCanBeTightenedInspection.java:101]: Looks like this is from 2020.1

**Comment by sfragata** [comment]
@robert3005 now the build got 74 tests 31 failed tests

**Comment by robert3005** [comment]
thanks @sfragata I generally use projects CI to test things out. It appears that this is not a matter of fixing compliation errors and there's something that changed with intellij internals

**Comment by robert3005** [comment]
I think this might need more updates that just what I am doing here. The tests run with latest changes, however, what's happening now is that classloaders are swapped upon loading components so the SaveActionManager that components get acccess to and the ones that's then available is a different object which then leads to test failures since it's not configured. If anyone wants to jump in go ahead but I don't see solutions other than converting to service/listeners but I only developed plugins in the post component intellij world so there might be a hack to get things working here.

**Comment by fishermans** [comment]
I am facing further issues. Hence, I created a new RR #392  which also includes the change mentioned here.

**Comment by dubreuia** [comment]
@robert3005 Thanks a lot for you work on the plugin! See #392 for resolution

**Pull request closed without merging**